wormhole-connect/src/sdk/index.ts

```diff
@@ -164,32 +170,45 @@ export const sendTransfer = async (
       toAddress,
       undefined,
     );
-    if (fromChainId !== MAINNET_CHAINS.solana) {
+    if (fromChainId === MAINNET_CHAINS.solana) {
+      const solTx = await signSolanaTransaction(
+        tx as Transaction,
+        TransferWallet.SENDING,
+      );
+      wh.registerProviders();
+      return solTx;
+    } else if (fromChainId === MAINNET_CHAINS.sui) {
+      const response = await signSuiTransaction(
+        tx as unknown as TransactionBlock,
+        TransferWallet.SENDING,
+      );
+      wh.registerProviders();
+      return response;
+    } else {
```

Comment:
We should figure out a system here.  It's going to get bloated as we add in more contexts

Reply:
Agreed, we will refactor this in the future.